param.in.sample

```diff
@@ -3,11 +3,11 @@
 )---------------------------------------------------------------------
 ) Important integration parameters:
 )---------------------------------------------------------------------
- algorithm (MVS, BS, BS2, RADAU, HYBRID etc) = bs
- start time (days)= 2451000.5
- stop time (days) = 2501000.5
- output interval (days) = 365.25d0
- timestep (days) = 8
+ algorithm (MVS, BS, BS2, RADAU, HYBRID etc) = radau
+ start time (days)= 2454441.5d0
+ stop time (days) = 2501000.5d0
+ output interval (days) = 101.d0
```

Comment:
As i said above, those values can be freely altered depending on what we want to get in output.